src/Atmos/Model/AtmosModel.jl

```diff
@@ -237,16 +237,23 @@ end
     vars_state(m::AtmosModel, ::Prognostic, FT)
 
 Conserved state variables (prognostic variables).
+
+!!! warning
+
+    The order of the fields for `AtmosModel` needs to match the one for
+    `AtmosLinearModel` since a shared state is used
```

Comment:
There is a rebase conflict

Reply:
just rebased

Reply:
This code already exists in master, so I think something is still wrong.

Reply:
I will close this one and do a new PR